### PR TITLE
[wasm] Add a --il-strip option to strip IL code from assemblies in AOT mode.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: sdks/wasm/packager.cs [hunk 1]
```diff
@@ -610,6 +619,9 @@ class WasmOptions {
 		ninja.WriteLine ("  description = [IL-LINK]");
 		ninja.WriteLine ("rule aot-dummy");
 		ninja.WriteLine ("  command = echo > aot-dummy.cs; csc /out:$out /target:library aot-dummy.cs");
+		ninja.WriteLine ("rule ilstrip");
+		ninja.WriteLine ("  command = cp $in $out; mono-cil-strip $out");
```
Comment (kjpou1) [inline sdks/wasm/packager.cs]: Where is `mono-cil-strip`?

Comment (kjpou1) [reply sdks/wasm/packager.cs]: Shouldn't the `--il-strip` be added to the list of options as well?

Comment (vargaz) [reply sdks/wasm/packager.cs]: It is added by this line:
		AddFlag (p, new BoolFlag ("il-strip", "strip IL code from assemblies in AOT mode", opts.ILStrip, b => opts.ILStrip = b));

Comment (vargaz) [reply sdks/wasm/packager.cs]: mono-cil-strip is an existing mono tool.

Comment (kjpou1) [reply sdks/wasm/packager.cs]: Yes see that now.  I thought it was part of the `wasm-tools`.

**Comment by kjpou1** [comment]
@monojenkins build failed

**Pull request merged**